### PR TITLE
Modernize Stat Value Playbook Kit

### PR DESCRIPTION
**Pull request by kskilt**
Common modernization to the Stat Value kit to match our current formatting.

Additional Information:
This kit originally accepted strings or integers where it was displaying values. The values were always an integer, but it was formatted this way in the examples if the end user typed in a string. To better stream line, we made the prop always a number, however this may effect previous uses that contained strings. With that in mind, we should watch any previous Stat Value pbkit or Dashboard Value kit (which inherits Stat Value in it) for backwards compatibility issues. It will be an easy fix of adjusting the value to an integer.

### CODE REVIEW

File: app/pb_kits/playbook/pb_stat_value/stat_value.rb [hunk 1]
```diff
@@ -2,61 +2,17 @@
...
-        "pb_stat_value_kit"
-      end
-
-      def to_partial_path
-        "pb_stat_value/stat_value"
-      end
+      partial "pb_stat_value/stat_value"
 
-    private
+      prop :unit
```
Comment (gillianwenhold) [inline app/pb_kits/playbook/pb_stat_value/stat_value.rb]: Does this one need a `Playbook::Props::String` and `default: nil`?

Comment (garettarrowood) [reply app/pb_kits/playbook/pb_stat_value/stat_value.rb]: He could have specified these. But since they are the default, it's okay to leave them off.

Comment (kskilt) [reply app/pb_kits/playbook/pb_stat_value/stat_value.rb]: So for a string prop, it is the default and automatically assumed by the code. No need to specify that or a default as nil.

**Comment by kskilt** [comment]
> No added spec file???
@KFad11 
End of the week vibes 😄 Added spec file now.

**Pull request merged**